go.mod

```diff
@@ -215,6 +215,7 @@ require (
 	github.com/spf13/pflag v1.0.5 // indirect
 	github.com/stretchr/objx v0.5.2 // indirect
 	github.com/tmc/grpc-websocket-proxy v0.0.0-20220101234140-673ab2c3ae75 // indirect
+	github.com/ulikunitz/xz v0.5.12 // indirect
```

Comment:
xz :expressionless:

Reply:
can easily replace, what would you prefer instead?

Reply:
i'm not sure either, maybe zstd:?

Reply:
When choosing I also try to use a format which users could potentially go to the file system and open extract the logs easily. `xz` was compressing our logs a bit better than `gz`, so I simply picked it (and since the backdoor story is already over). I am fully open to suggestions - can even go with gzip.

Reply:
I would propose to take a look at https://github.com/klauspost/compress, this has way better compression methods.

Let's skip xz and the library

Reply:
will do